cypress/e2e/opening-hours-editor.cy.ts

```diff
@@ -0,0 +1,298 @@
+const branchTitle = "Test branch";
+const formattedSearchString = branchTitle.toLowerCase().replace(/ /g, "+");
+
+enum OpeningHourCategories {
+  Opening = "Åbent",
+  CitizenService = "Borgerservice",
+  WithService = "Med betjening",
+  SelfService = "Selvbetjening",
+  PhoneTime = "Telefontid",
+}
+
+type OpeningHourFormType = {
+  openingHourCategory: OpeningHourCategories;
+  start: `${number}:${number}`;
+  end: `${number}:${number}`;
+};
+
+const createTestBranchAndVisitOpeningHoursAdmin = () => {
+  cy.drupalLoginAndVisit("/node/add/branch");
+  cy.get("#edit-title-0-value").type(branchTitle);
+  cy.get('button[title="Show all Paragraphs"]').click();
+  cy.get('button[value="Opening Hours"]').click({
+    multiple: true,
+    force: true,
+  });
+  cy.get("#edit-field-address-0-address-address-line1")
+    .type("Example Street", { force: true })
+    .should("have.value", "Example Street");
+  cy.get("#edit-field-address-0-address-postal-code")
+    .type("1234", { force: true })
+    .should("have.value", "1234");
+  cy.get("#edit-field-address-0-address-locality")
+    .type("Example City", { force: true })
+    .should("have.value", "Example City");
+  cy.get('input[value="Save"]').click();
+  cy.get('a[href^="/node/"][href$="/edit"]').click({ force: true });
+  cy.get('a[href*="/edit/opening-hours"]').click();
+  // Save the URL for the admin page and the page itself for later use
+  cy.url().then((url) => {
+    Cypress.env("adminUrl", url);
+    const pageUrl = url.replace("/edit/opening-hours", "");
+    Cypress.env("pageUrl", pageUrl);
+  });
```

Comment:
This creates a hidden dependency between functions `createTestBranchAndVisitOpeningHoursAdmin()`, `visitOpeningHoursPage()`, `visitOpeningHoursPageAdmin()`. I am not sure if anything can be done to address this at this point though.